src/data/ons_pd_may_19.py

```diff
@@ -14,21 +14,23 @@ class ONSPostcodeDirectoryMay19(ONSPostcodeDirectory):
     :var dict ONSPostcodeDirectoryMay19.IMD_MAX: Highest ranked Lower Level Super Output Area (or equivalent) in each country
     :var dict ONSPostcodeDirectoryMay19.COUNTRY_CODES: ONS Postcode Directory codes for each country
     """
-    fields = ['lsoa11', 'msoa11', 'oslaua', 'osward', 'pcon', 'oscty', 'oseast1m', 'osnrth1m', 'lat', 'long', 'imd', 'ctry', 'rgn', 'pcd']
-    index_column = 'pcd'
+    # fields = ['lsoa11', 'msoa11', 'oslaua', 'osward', 'pcon', 'oscty', 'oseast1m', 'osnrth1m', 'lat', 'long', 'imd', 'ctry', 'rgn', 'pcd']
```

Comment:
Either explain why these comments are useful or remove them?

Reply:
This is for the same reason as below - something I'm testing but haven't fully implemented.

Basically the full lat/long + easting/northing are needed on merging with the census dataset, but for geography transform operations we're pretty much using the ONS dataset to map some geographies to others - by removing these columns and dropping duplicates I'm able to get the ONS dataset to 0.35% of its original size, better for memory consumption etc.

See `scripts/setup_reduce_onspd.py` for the reduction code (64a2ab7)